config/config.go

```diff
@@ -1,20 +1,47 @@
 package config
 
 import (
+	"crypto/x509"
+	"encoding/pem"
+	"log"
+	"os"
+
 	"github.com/Netflix/go-env"
 )
 
 type Config struct {
 	GrpcListenAddress string `env:"GRPC_LISTEN_ADDRESS,default=0.0.0.0:8080"`
 	SQLiteDirPath     string `env:"SQLITE_DIR_PATH,default=db"`
 	PgDatabaseUrl     string `env:"DATABASE_URL"`
+	CACertPath        string `env:"CA_CERT_PATH"`
+	CACert            *x509.Certificate
+}
+
+func initializeCACert(certPath string) *x509.Certificate {
+	certData, err := os.ReadFile(certPath)
+	if err != nil {
```

Comment:
Worth I think add a flag to the config to skip API key validation. I am thinking about running it locally during development.